channels/us.m3u

```diff
@@ -6396,7 +6370,19 @@ https://ln-zen.localnowlive.com/v1/master/385c85a93929f94966d0fb186fc33b431e6f1e
 https://rockentertainment-zoomoo-1.samsung.wurl.com/manifest/playlist.m3u8
 #EXTINF:-1 tvg-id="KanalDisney.us" tvg-name="Канал Disney" tvg-country="RU" tvg-language="Russian" tvg-logo="https://i.imgur.com/Q9KoVy9.png" group-title="Kids",Канал Disney (576p)
 http://188.40.68.167/russia/disney/playlist.m3u8
+#EXTINF:-1 tvg-id="Dabl.us" tvg-name="Dabl" tvg-country="US" tvg-language="English" tvg-logo="" group-title="",Dabl (1080p)
+http://dai.google.com/linear/hls/event/oIKcyC8QThaW4F2KeB-Tdw/master.m3u8
 #EXTINF:-1 tvg-id="KABCTV.us" tvg-name="ABC 7 Los Angeles CA (KABC-TV)" tvg-country="US" tvg-language="English" tvg-logo="https://imgur.com/nKAUCLF.jpg" group-title="News",ABC 7 Los Angeles CA (KABC-TV)
 https://content.uplynk.com/channel/ext/2118d9222a87420ab69223af9cfa0a0f/KABCwatch.m3u8
 #EXTINF:-1 tvg-id="KSTPTV.us" tvg-name="ABC 5 Minneapolis MN (KSTP)" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/UZWNjGs.png" group-title="News",ABC 5 Minneapolis MN (KSTP)
 https://iptv-all.lanesh4d0w.codes/united-states/kstp-tv
+#EXTINF:-1 tvg-id="PlutoTVTeenNick.us" tvg-name="Pluto TV TeenNick" tvg-country="US" tvg-language="English" tvg-logo="" group-title="Kids",Pluto TV TeenNick
+http://stitcher-ipv4.pluto.tv/v1/stitch/embed/hls/channel/5f0d668b872e4400073acc68/master.m3u8?deviceType=samsung-tvplus&deviceMake=samsung&deviceModel=samsung&deviceVersion=unknown&appVersion=unknown&deviceLat=0&deviceLon=0&deviceDNT=%7BTARGETOPT%7D&deviceId=%7BPSID%7D&advertisingId=%7BPSID%7D&us_privacy=1YNY&samsung_app_domain=%7BAPP_DOMAIN%7D&samsung_app_name=%7BAPP_NAME%7D&profileLimit=&profileFloor=&embedPartner=samsung-tvplus
```

Comment:
```suggestion
http://stitcher-ipv4.pluto.tv/v1/stitch/embed/hls/channel/5f0d668b872e4400073acc68/master.m3u8?serverSideAds=false&deviceType=samsung-tvplus&deviceMake=samsung&deviceModel=samsung&deviceVersion=unknown&appVersion=unknown&deviceLat=0&deviceLon=0&deviceDNT=%7BTARGETOPT%7D&deviceId=%7BPSID%7D&advertisingId=%7BPSID%7D&us_privacy=1YNY&samsung_app_domain=%7BAPP_DOMAIN%7D&samsung_app_name=%7BAPP_NAME%7D&profileLimit=&profileFloor=&embedPartner=samsung-tvplus
```

Disables server-sided advertisments to prevent stream cutoffs.

Reply:
Changed URL

Reply:
Added new link.